images/base/Dockerfile

```diff
@@ -12,6 +12,8 @@ FROM centos:centos7
 # base images land
 RUN yum swap -y -- remove systemd-container\* -- install systemd systemd-libs
 
-RUN yum install -y git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
+RUN yum install --setopt=tsflags=nodocs -y git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
     socat ethtool device-mapper iptables && \
-    yum clean all
+    yum clean all && \
+    rpm -V git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
```

Comment:
@sdodson actually if we are squashing the images, the layer will be squashed anyway... but the list of packages will appear in the image metadata... also you will have problem if you run yum install twice (the env will override the parent env..)